bugfix/klaviyo-too-many-partitions (#26)
fivetran/dbt_shopify_holistic_reporting: CHANGELOG.md

# dbt_shopify_holistic_reporting v0.7.0

[PR #26](https://github.com/fivetran/dbt_shopify_holistic_reporting/pull/26) includes the following changes:

## Upstream Klaviyo Breaking Changes (Full refresh required after upgrading)

> - Upstream incremental models within the dbt_klaviyo package running on BigQuery have had the `partition_by` logic removed from incremental models running on BigQuery. This change affects only BigQuery warehouses and resolves the `too many partitions` error that some users encountered. The partitioning was also deemed unnecessary for the mentioned models and their downstream references, offering no performance benefit. By removing it, we eliminate both the error risk and an unneeded configuration. Refer to the [v0.8.0 dbt_klaviyo release notes](https://github.com/fivetran/dbt_klaviyo/releases/tag/v0.7.2) for more details. This change applies to the following models:

Reviewer: ```suggestion
- Upstream incremental models within the dbt_klaviyo package running on BigQuery have had the `partition_by` logic removed from incremental models running on BigQuery. This change affects only BigQuery warehouses and resolves the `too many partitions` error that some users encountered. The partitioning was also deemed unnecessary for the mentioned models and their downstream references, offering no performance benefit. By removing it, we eliminate both the error risk and an unneeded configuration. Refer to the [v0.8.0 dbt_klaviyo release notes](https://github.com/fivetran/dbt_klaviyo/releases/tag/v0.8.0) for more details. This change applies to the following models:
```

Author: Thanks for catching this!

> - Upstream incremental models within the dbt_klaviyo package running on BigQuery have had the `partition_by` logic removed from incremental models running on BigQuery. This change affects only BigQuery warehouses and resolves the `too many partitions` error that some users encountered. The partitioning was also deemed unnecessary for the mentioned models and their downstream references, offering no performance benefit. By removing it, we eliminate both the error risk and an unneeded configuration. Refer to the [v0.8.0 dbt_klaviyo release notes](https://github.com/fivetran/dbt_klaviyo/releases/tag/v0.8.0) for more details. This change applies to the following models:

Reviewer 2: ```suggestion
- Upstream incremental models within the dbt_klaviyo package running on BigQuery have had the `partition_by` logic removed. This change affects only BigQuery warehouses and resolves the `too many partitions` error that some users encountered. The partitioning was also deemed unnecessary for the aforementioned models and their downstream references, offering no performance benefit. By removing it, we eliminate both the error risk and an unneeded configuration. Refer to the [v0.8.0 dbt_klaviyo release notes](https://github.com/fivetran/dbt_klaviyo/releases/tag/v0.8.0) for more details. This change applies to the following models:
```

  - `int_klaviyo__event_attribution`
  - `klaviyo__events`

## Under the Hood

> - Added consistency and integrity validation tests for the following models:

Reviewer 2: ```suggestion
- Added consistency validation tests for the following models:
```

# dbt_shopify_holistic_reporting v0.7.0

[PR #26](https://github.com/fivetran/dbt_shopify_holistic_reporting/pull/26) includes the following changes:

## Upstream Klaviyo and Shopify Holistic Reporting Breaking Changes (Full refresh required after upgrading)

> - Upstream and immediate incremental models within the dbt_klaviyo and dbt_shopify_holistic_reporting package running on BigQuery have had the `partition_by` logic removed. This change affects only BigQuery warehouses and resolves the `too many partitions` error that some users encountered. The partitioning was also deemed unnecessary for the aforementioned models and their downstream references, offering no performance benefit. By removing it, we eliminate both the error risk and an unneeded configuration. Refer to the [v0.8.0 dbt_klaviyo release notes](https://github.com/fivetran/dbt_klaviyo/releases/tag/v0.8.0) for more details regarding the upstream changes. This change applies to the following models:

Reviewer 2: ```suggestion
- Upstream and immediate incremental models within the dbt_klaviyo and dbt_shopify_holistic_reporting packages running on BigQuery have had the `partition_by` logic removed. This change affects only BigQuery warehouses and resolves the `too many partitions` error that some users encountered. The partitioning was also deemed unnecessary for the aforementioned models and their downstream references, offering no performance benefit. By removing it, we eliminate both the error risk and an unneeded configuration. Refer to the [v0.8.0 dbt_klaviyo release notes](https://github.com/fivetran/dbt_klaviyo/releases/tag/v0.8.0) for more details regarding the upstream changes. This change applies to the following models:
```
